CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Head
 
 - Fixed: `block-*-brace-*-before` CRLF (`\r\n`) warning positioning.
+- Fixed: `selector-no-type` and `selector-type-case` now ignore non-standard keyframe selectors (e.g. within an SCSS mixin).
+- Fixed: `selector-type-no-unknown` no longer reports factional keyframe selectors.
```

Comment:
Done in https://github.com/stylelint/stylelint/commit/d937d7262b5f78e350741da9c35c746271185754.